apps/portal/app/components/create-identity/create-identity-form.tsx

```diff
@@ -310,6 +311,20 @@ export function IdentityForm({
           const receipt = await publicClient.waitForTransactionReceipt({
             hash: txHash,
           })
+          const decodedLog = decodeEventLog({
+            abi: multivaultAbi,
+            data: receipt?.logs[0].data,
+            topics: receipt?.logs[0].topics,
+          })
+
+          console.log('decodedLog', decodedLog)
```

Comment:
Let's remove this or change to use `logger`

Reply:
Good callout, completely forgot I added that